TestPullRequest/src/testPackage/Calc.java

```diff
@@ -4,7 +4,7 @@
 	
 	public final static int MULTIPLIER = 10;
 
-	public Integer addition(int a, int b, int c) {
+	public Integer testNewID(int a, int b, int c) {
```

Comment:
@LHommeDeBot: Could you please rename this method to something like 'testRegex'? You can contact me at 'mymail@hotmail.com' or check 'www.google.de'!